src/Xamarin.Android.Tools.AndroidSdk/Sdks/AndroidSdkBase.cs

```diff
@@ -65,7 +65,7 @@ public virtual void Initialize (string androidSdkPath = null, string androidNdkP
 			androidNdkPath  = androidNdkPath ?? PreferedAndroidNdkPath;
 			javaSdkPath     = javaSdkPath ?? PreferedJavaSdkPath;
 
-			AndroidSdkPath  = ValidateAndroidSdkLocation (androidSdkPath) ? androidSdkPath : AllAndroidSdks.FirstOrDefault ();
+			AndroidSdkPath  = androidSdkPath ?? AllAndroidSdks.FirstOrDefault ();
```

Comment:
Does this line need to change? The net effect of this looks like we just removed validation of the supplied path

Reply:
Yeah, the idea of the PR is to remove the validation of `PreferredAndroidSdkPath`. Otherwise sdk-manager wouldn't know where to install android sdk if it is absent.